### PR TITLE
fix error sending discord webhooks

### PR DESCRIPTION
**Pull request by Razzmatazzz**

### CODE REVIEW

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Review by GrantBirki — APPROVED** [review]

File: src/tarkov-data-manager/jobs/check-scans.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ class CheckScansJob extends DataJob {
             };
 
             this.logger.log('Sending alert');
-            await this.discordAlert(messageData);
+            this.discordAlert(messageData);
```
Comment (GrantBirki) [inline src/tarkov-data-manager/jobs/check-scans.js:68]: I'm still a n00b around all the `async` and `await` stuff in JS. @Razzmatazzz why did removing the `await` statement here fix things?

Comment (Razzmatazzz) [reply src/tarkov-data-manager/jobs/check-scans.js:68]: That particular removal didn't fix anything. The fix was updating our discord webhook code to work with the updated discord.js library. I also changed the discord webhook function of the DataJob class (which CheckScansJob extends) to ensure that all webhooks are awaited before the job completes. That allows individual webhooks to be sent in the middle of a job asynchronously without having to wait for it to resolve so the job can continue to execute.

In short, I removed this particular await because it's no longer necessary and not using it is more consistent with best practices in other jobs.

**Review by Shebuka — APPROVED** [review]
